test/test_draw.py

```diff
@@ -37,6 +42,9 @@ def testPointInImageDimensions(self):
         print('\nPoints can be drawn as the dimentions are in the range of the output image size: PASS.')
 
     def testPointOutOfImageDimensions(self):
+        
+        """" Checking the points  which are out of range """
```

Comment:
don't use docstrings inplace of comments.
comments are comments and should start with #.
docstrings are only used to describe modules, classes, types, functions not as comments.

Same everywhere inside the test files.